.github/renovate.json5

```
@@ -0,0 +1,45 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:recommended',
+    'default:pinDigestsDisabled',
+    'security:openssf-scorecard'
```

Can you please add https://docs.renovatebot.com/presets-default/#semanticcommits ?

```suggestion
    'security:openssf-scorecard',
    ':semanticCommits'
  ],
```